app/containers/dock-dx.js

```diff
@@ -0,0 +1,98 @@
+import React, { cloneElement, Children, Component, PropTypes } from 'react';
+import Dock from 'react-dock';
+import parseKey from 'parse-key';
+import JSONTree from 'react-json-tree'
+
+export default class DockDX extends Component {
```

Comment:
I think When they will be included in focus-components/Dev-tools  or in a focus-devtools repository then in the import I am ok with you